src/app/components/ImageCarousel/ImageCarousel.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Carousel } from 'react-responsive-carousel';
+import Image from '../Image/Image';
+
+import 'react-responsive-carousel/lib/styles/carousel.min.css';
+
+type imageCarouselProps = {
+  showArrows: boolean;
+  image: string[];
+};
+
+export default function ImageCarousel({
+  showArrows,
+  image,
+}: imageCarouselProps): JSX.Element {
+  return (
+    <Carousel showArrows={showArrows}>
+      {image.map((URL) => (
+        <Image imageURL={URL} />
```

Comment:
```suggestion
      {image.map((url) => (
        <Image imageURL={url} />
```
Capitalized constants normally describe a never changing constant, so these should not be capitalized